### PR TITLE
OSDOCS-4439: Adding info on mDNS in MicroShift

### PR DESCRIPTION
**Pull request by ShaunaDiaz**
[OSDOCS-4439](https://issues.redhat.com//browse/OSDOCS-4439): Adds information on mDNS in MicroShift to Networking

Version(s):
4.12

Issue:
https://issues.redhat.com/browse/OSDOCS-4439

Link to docs preview:
https://53878--docspreview.netlify.app/microshift/latest/microshift_networking/microshift-networking.html#microshift-mDNS_microshift-networking

QE review:
N/A - MicroShift is Developer Preview
SME approval is required

### CODE REVIEW

**Review by mangelajo — COMMENTED** [review]
/lgtm

**Comment by skrthomas** [comment]
@ShaunaDiaz can you link to the preview? TY!

**Comment by ShaunaDiaz** [comment]
@skrthomas Yes, sorry, 'tis there in comments above now.

File: modules/microshift-mDNS.adoc [hunk 1]
```diff
@@ -0,0 +1,12 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-networking.adoc
+
+:_content-type: CONCEPT
+[id="microshift-mDNS_{context}"]
+== mDNS
```
Comment (skrthomas) [inline modules/microshift-mDNS.adoc:7]: If this is the first mention of mDNS, it shouldn't be abbreviated, per IBMSG guidance on abbreviations.  Multicast DNS (mDNS)

Comment (skrthomas) [inline modules/microshift-mDNS.adoc:7]:
```suggestion
== The multicast DNS Protocol
```

File: modules/microshift-mDNS.adoc [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+mDNS is a protocol used to allow name resolution and service discovery within a Local Area Network (LAN) using multicast exposed on the `5353/UDP` port.
```
Comment (skrthomas) [inline modules/microshift-mDNS.adoc:8]:
```suggestion
The multicast DNS protocol (mDNS) allows name resolution and service discovery within a Local Area Network (LAN) using multicast exposed on the `5353/UDP` port.
```

File: modules/microshift-mDNS.adoc [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+{product-title} includes an embedded mDNS server for deployment scenarios in which the authoritative DNS server cannot be reconfigured to point clients to services on {product-title}.
+
+This allows `.local` domains exposed by {product-title} to be discovered by other elements on the LAN.
```
Comment (skrthomas) [inline modules/microshift-mDNS.adoc:12]: "This allows `.local`..." seems a little orphaned conceptually :/ can you incorporate it more with the preceding paragraph so the "this" isn't questionable?

**Review by skrthomas — COMMENTED** [review]
A few comments for ya :)

**Pull request merged**